articles/references/domain-designer/associations/associations.md

```diff
@@ -3,36 +3,34 @@ uid: references.domain-designer.associations
-Wikipedia provides some information regarding the concept of [Relationships](https://en.wikipedia.org/wiki/Class_diagram#Relationships) if you want to find out more.
+Intent Architect's Domain Designer enables modelling of a sub-set of [UML](https://en.wikipedia.org/wiki/Unified_Modeling_Language), in particular, [class diagrams](https://en.wikipedia.org/wiki/Class_diagram). It supports modelling of Classes, their Attributes, Operations (method invocations) and [Associations (relationships)](https://en.wikipedia.org/wiki/Class_diagram#Relationships) between classes. Additionally, [Stereotypes](https://en.wikipedia.org/wiki/Stereotype_(UML)) can be used to extend the vocabulary of elements on the diagram.
 
-In the UML world, an `Association` is a kind of `Relationship` while in Intent Architect's Domain Designer the term `Association` represents the kind of relationship between two `Entities` (or `Classes`). 
+> [!NOTE]
+> For more information on using Stereotypes in Intent Architect you can refer to [this article](xref:how-to-guides.use-stereotypes).
 
-The following information around associations can be modeled in Intent Architect:
+This article will focus on the different options available within the Domain Designer for modelling Associations and how to do so.
 
-- [Association Types](xref:references.domain-designer.associations#association-types) - Relationship ownership between Entities
-- [Multiplicity Types](xref:references.domain-designer.associations#multiplicity-types) - Cardinality between related Entities
-- [Navigability](xref:references.domain-designer.associations#navigability) - Constraining the direction of accessing Entities from other Entities
+> [!NOTE]
+> This article will regularly refer to `Source End` and `Target End`, these are sections in the properties pane in the bottom-right corner of the Domain Designer:
+>
+> ![The properties pane in Intent Architect's Domain Designer](images/properties-pane.png)
 
-## Association Types
+## Association types
 
->[!NOTE]
->Reference to the `Source End` and `Target End` properties will be made in this document. You will find this in Intent Architect
->when you locate the Properties panel on the screen to the right.
+Association types are used to describe the ownership between Entities.
 
 ### Composition Association
 
->Composition relationship
+> Composition relationship
 >
->1. When attempting to represent real-world whole-part relationships, e.g. an engine is a part of a car.
->2. When the container is destroyed, the contents are also destroyed, e.g. a university and its departments.
-
-*[Wikipedia - Composition](https://en.wikipedia.org/wiki/Class_diagram#Composition)*
+> 1. When attempting to represent real-world whole-part relationships, e.g. an engine is a part of a car.
+> 2. When the container is destroyed, the contents are also destroyed, e.g. a university and its departments.
+>
+> *- [Wikipedia](https://en.wikipedia.org/wiki/Class_diagram#Composition)*
```

Comment:
I feel it's fine and nicer looking to just mention the source of the quote, no need to mention the title of the source.